sdk/storage/storage-blob-changefeed/package.json

```diff
@@ -0,0 +1,163 @@
+    "execute:js-samples": "node ../../../common/scripts/run-samples.js samples/javascript/",
+    "execute:ts-samples": "node ../../../common/scripts/run-samples.js samples/typescript/dist/samples/typescript/src/",
+    "execute:samples": "npm run build:samples && npm run execute:js-samples && npm run execute:ts-samples",
+    "format": "prettier --write --config ../../.prettierrc.json \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
+    "integration-test:browser": "karma start --single-run",
+    "integration-test:node": "nyc mocha -r esm --require source-map-support/register --reporter ../../../common/tools/mocha-multi-reporter.js --full-trace -t 300000  dist-esm/storage-blob-changefeed/test/*.spec.js dist-esm/storage-blob-changefeed/test/node/*.spec.js",
+    "integration-test": "npm run integration-test:node && npm run integration-test:browser",
+    "pack": "npm pack 2>&1",
+    "prebuild": "npm run clean",
+    "test:browser": "npm run clean && npm run build:test && npm run unit-test:browser",
+    "test:node": "npm run clean && npm run build:test && npm run unit-test:node",
+    "test": "npm run clean && npm run build:test && npm run unit-test",
+    "unit-test:browser": "karma start --single-run",
+    "unit-test:node": "mocha --require source-map-support/register --reporter ../../../common/tools/mocha-multi-reporter.js --full-trace -t 120000 dist-test/index.node.js",
+    "unit-test": "npm run unit-test:node && npm run unit-test:browser",
+    "emulator-tests": "cross-env STORAGE_CONNECTION_STRING=UseDevelopmentStorage=true && npm run test:node"
+  },
+  "files": [
+    "BreakingChanges.md",
+    "types/",
+    "dist/",
+    "dist-browser/",
+    "dist-esm/storage-blob-changefeed/src/",
+    "dist-esm/storage-internal-avro/src/",
+    "typings/latest/storage-blob-changefeed.d.ts",
+    "typings/3.1/storage-blob-changefeed.d.ts",
+    "README.md",
+    "LICENSE"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Azure/azure-sdk-for-js.git"
+  },
+  "keywords": [
+    "Azure",
+    "Storage",
+    "Blob",
+    "Change feed",
+    "Node.js",
+    "TypeScript",
+    "JavaScript",
+    "Browser"
+  ],
+  "author": "Microsoft Corporation",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/Azure/azure-sdk-for-js/issues"
+  },
+  "homepage": "https://github.com/Azure/azure-sdk-for-js#readme",
+  "sideEffects": false,
+  "//metadata": {
+    "constantPaths": [
+      {
+        "path": "src/utils/constants.ts",
+        "prefix": "SDK_VERSION"
+      }
+    ]
+  },
+  "dependencies": {
+    "@azure/storage-blob": "^12.1.2",
+    "@azure/abort-controller": "^1.0.0",
+    "@azure/core-http": "^1.1.1",
+    "@azure/core-lro": "^1.0.2",
+    "@azure/core-paging": "^1.1.1",
+    "@azure/core-tracing": "1.0.0-preview.8",
+    "@azure/logger": "^1.0.0",
+    "@opentelemetry/api": "^0.6.1",
+    "events": "^3.0.0",
+    "tslib": "^1.10.0"
+  },
+  "devDependencies": {
```

Comment:
>devDependencies [](start = 3, length = 15)

Not sure all of the dependencies are required here. Please clean up when GA the package.